Resources/Maps/northway.yml

```
@@ -3600,15 +3609,15 @@
     - type: Transform
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 6 but found 4

(indentation)